composer.json

```diff
@@ -30,7 +30,8 @@
         "doctrine/lexer": "^1.0",
         "doctrine/persistence": "^2.2",
         "psr/cache": "^1 || ^2 || ^3",
-        "symfony/console": "^3.0|^4.0|^5.0|^6.0"
+        "symfony/console": "^3.0|^4.0|^5.0|^6.0",
+        "symfony/polyfill-php80": "^1.15"
```

Comment:
Note: I did this to gain access to the `get_debug_type()` function. The polyfill is already a dependency of `symfony/console`, so I'm effectively not introducing a new dependency.

Reply:
* of `symfony/console` 5.3 :stuck_out_tongue:

Reply:
Console 4.4 also pulls the polyfill: https://packagist.org/packages/symfony/console#v4.4.25